service/clientsrv/clientsrv.go

```diff
@@ -1,4 +1,4 @@
-package replica
+package clientsrv
```

Comment:
This suggestion might be controversial. Maybe even this `clientsrv` package can be moved into the current `clientpb` package (assuming that we rename it to `mempool`). Then this file would be named `mempool_server.go` and the struct would be accessed as `mempool.Server` and `mempool.New` or `mempool.NewServer`. (see also my comments in `cmdcache.go`)